core/modules/system/image.gd.inc

```diff
@@ -58,15 +57,31 @@ function image_gd_check_settings() {
  *   The new width of the resized image, in pixels.
  * @param $height
  *   The new height of the resized image, in pixels.
+ * @param $background
+ *   A hexadecimal integer specifying the background color to use for the
+ *   uncovered area of the image after transformation. Examples: "#RGB",
+ *   "#RGBA", "#RRGGBB", "#RRGGBBAA". For images that support transparency, this
+ *   will default to transparent. Otherwise it will be white (#FFFFFF)..
```

Comment:
These examples should exclude the hash (#), as $background will be invalid if it is included.

There's also an extra period at the end.